### ISSUE TITLE
DocumentHandler Editor crashes upon opening v1 config files

### ISSUE THREAD

**Issue by jacobwod** [hajkmap/Hajk#700]
**Disclaimer: I'm not sure what's been said about backwards compatibility between the new DocumentHandler and the v1 format of JSON files (from Informative).** 

If (and only _if_ - please see disclaimer) we aim for easy conversion between Informative and DocumentHandler, we must fix [this line](https://github.com/hajkmap/Hajk/blob/59e867e1f394662a87e326a5fe099eeb3491cd99/new-admin/src/views/components/DocumentTextEditor.jsx#L919) that currently leads to crash in admin:
![Skärmavbild 2021-03-12 kl  11 36 37](https://user-images.githubusercontent.com/110222/110928478-36b92a80-8327-11eb-8a36-252e269093d0.png)

```javascript
// Current
 <div style={editorContainer} onClick={this.focus}>
// Loads (but will obviously lack required formatting)
 <div onClick={this.focus}>
```

Just some checking on the contents of `editorContainer` will fix this.

**Comment by persoderstrom** [comment]
**Kungsbacka has to be able to easily convert from the old Informative to the new DocumentHandler format.**

**Comment by jacobwod** [comment]
Is someone looking into this, @linusfj, @sweco-secoma?

**Comment by sweco-secoma** [comment]
It's SBK and @linusfj  that have been taking care of the Editor part, and I think Linus has a few changes that will be coming in - not sure if this is one of them. I'll unassign myself here as it's not something we're working with as it stands.

**Comment by jacobwod** [comment]
@perosoderstrom, I needed to get rid of this so I just made a super-quick fix (7b997c1) that gets my old config files to load. Feel free to check out if this solves your loading issues. 

NB: This editor is still full of quirks but I did no attempt to fix them.
